src/SourceBuild/content/test/Microsoft.DotNet.Installer.Tests/LinuxInstallerTests.cs

```diff
@@ -53,13 +54,15 @@ public LinuxInstallerTests(ITestOutputHelper outputHelper)
         Directory.CreateDirectory(_tmpDir);
         _contextDir = Path.Combine(_tmpDir, Path.GetRandomFileName());
         Directory.CreateDirectory(_contextDir);
+
+        _excludeLinuxArch = Config.Architecture == Architecture.X64 ? "arm64" : "x64";
```

Comment:
Would it be wrong to use the tolower string representation of the enum?

Reply:
No, it would be best to do that - will fix.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/47313/commits/d9895e41c6e5fd6481b02d77f7dbe6a9bd7efec0